### DISCUSSION TITLE
defineRouting - Cannot find module 'next-intl/routing' or its corresponding type declarations.ts(2307)

### DISCUSSION THREAD

**Discussion by kliatskiidenis** [amannn/next-intl#1429] [General]
Hi, I have a project on Nextjs14.1.34 with APP routing, using “next-intl”: “3.14.0”,
The project is configured and working on the app, but I saw in the official documentation that the next-intl settings have changed a bit and the following module appeared: 
import {defineRouting} from 'next-intl/routing';
I tried it and got an error: Cannot find module 'next-intl/routing' or its corresponding type declarations.ts(2307)

I'm currently connecting like this:
<img width="758" alt="Знімок екрана 2024-10-18 о 11 23 45" src="https://github.com/user-attachments/assets/7b49a79e-e9c6-46d9-a5de-92a1ee292cd0">

Questions:
Do I need to replace my customization with defineRouting?
Why is this module not imported, what is the reason? 
Could you please tell me?

**Comment by amannn** [comment]
Hey! Sorry if this was confusing, `defineRouting` was added in [`next-intl@3.18`](https://github.com/amannn/next-intl/releases/tag/v3.18.0). I've just set up a blog post to be published early next week that goes over recent changes in `next-intl`, I think that should help to clarify this! For the time being, in case you're unsure, you don't have to change your routing configuration.
